apps/game_backend/lib/game_backend/curse_of_mirra/quests.ex

```diff
@@ -26,6 +26,24 @@ defmodule GameBackend.CurseOfMirra.Quests do
     Repo.get_by(Quest, config_id: quest_config_id)
   end
 
+  @doc """
+  Get a list of %Quest{} by the type field
+
+  ## Examples
+
+      iex>get_quest_by_config_id("bounty")
+      [%Quest{type: "bounty"}]
+
+  """
+  def get_quest_by_type(type) do
```

Comment:
```suggestion
  def list_quest_by_type(type) do
```

Reply:
Fixed! i also had to change the description